services/factorio-mod-portal/factorio-mod-portal.tester.js

```diff
@@ -0,0 +1,50 @@
+import {
+  isComposerVersion,
```

Comment:
`isComposerVersion` is a fairly specific validator for PHP package ranges. It looks like what you really want here is probably `isVPlusDottedVersionNClauses` or `isVPlusDottedVersionAtLeastOne`.